upgrade-notes/backend_20250131_220504.md

```diff
@@ -0,0 +1,10 @@
+#### replace custom webfont icons with tabler icons provided by ux_icons ([#3765](https://github.com/shopsys/shopsys/pull/3765))
+
+- all icons are mpw rendered by the `ux_icons` twig function provided by the `symfony/ux-icons` package
+    - see the documentation page [Icons](https://docs.shopsys.com/administration/icons) to learn how to render icons the right way
+- all your custom icons, rendered as `<i class="svg svg-{iconName}"></i>`, should be replaced with the `ux_icons` function
+- you can use upgrade command `bin/console upgrade:replace-obsolete-icons` to replace all custom icons in your project in twig templates
+    - check if all icons were replaced correctly and are displayed as expected (search for `svg-` in your codebase)
+- icons in JavaScript files should be replaced manually, see the documentation page [Icons](https://docs.shopsys.com/administration/icons) for more information
+- if necessary, improve CSS styles, so the SVG icons are displayed correctly
+- see #project-base-diff to update your project
```

Comment:
It would be nice to add some default note to our upgrade notes to remember to run `php bin/console ux:icons:lock ` so added icons in framework will be locked

Reply:
makes sense :+1:

Reply:
improved a bit in 59d14f26b9